parser_library/test/workspace/load_config_test.cpp

```diff
@@ -354,4 +354,102 @@ TEST(workspace, asm_options_invalid)
     ws.collect_diags();
     ASSERT_EQ(ws.diags().size(), 1U);
     EXPECT_EQ(ws.diags()[0].code, "W0002");
+}
+
+class file_proc_grps_asm : public file_proc_grps
+{
+public:
+    file_proc_grps_asm(std::string_view asm_opts_contents)
+        : file_proc_grps()
+        , proc_file(generate_proc_file(asm_opts_contents))
+
+    {}
+
+    const std::string& get_text() override { return proc_file; }
+
+    std::string proc_file;
+
+private:
+    std::string generate_proc_file(std::string_view asm_opts_contents)
+    {
+        return is_windows() ?
```

Comment:
I don't see any path separators in these inputs, so no need to have win/non-win version.